fdbserver/ApplyMetadataMutation.cpp

```diff
@@ -687,6 +688,13 @@ class ApplyMetadataMutationsImpl {
 					(*tenantNameIndex)[tenantEntry.tenantName] = tenantEntry.id;
 				}
 			}
+			if (lockedTenants) {
+				if (tenantEntry.tenantLockState == TenantLockState::LOCKED) {
```

Comment:
Should this also be checking for read-only locks?

Reply:
Yes you're right